codecov.yml

```diff
@@ -0,0 +1,13 @@
+# Documentation: https://docs.codecov.io/docs/codecov-yaml
+coverage:
+    status:
+        # Allows coverage to drop by a 3% when compared against the base commit.
+        project:
+            default:
+                target: auto
+                threshold: 3%
+        # Sets the expected status for `codecov/patch` check.
+        # We set this to be only informational hence it does not cause the check to fail.
+        patch:
+            default:
+                informational: true
```

Comment:
Do we not want to block? or maybe I don't know enough about this option?

Reply:
I think that we do not want to block on codecov because we've been hearing that codecov sometimes behaves weirdly for some PRs.

Reply:
Sometimes the difference calculated by codecov is 100% from the previous commit.
Informational only - this way we still have the info but it does not fail if the commit in the same PR has a difference of 100%.
Checking the coverage diff between commits (patch) as it tends to change when rebasing the code.

Reply:
Ack